src/main/scala/org/dia/algorithms/mcs/MCSUtils.scala

```diff
@@ -118,11 +123,13 @@ object MCSUtils {
       datafile.close()
     }
     catch {
-      case _: Throwable => logger.info("Error generating netCDF file for " + filepath + "\n")
+      case _: Throwable => logger.info("Error generating netCDF file for " + currFile + "\n")
```

Comment:
The error can be thrown for:
slicing the data or creating the file writer (https://github.com/SciSpark/SciSpark/blob/master/src/main/scala/org/dia/algorithms/mcs/MCSUtils.scala#L83:L85)
or writing the actual file 
https://github.com/SciSpark/SciSpark/blob/master/src/main/scala/org/dia/algorithms/mcs/MCSUtils.scala#L113:L118
refactor the try block to accommodate these in another PR.